cmd/aws-iam-authenticator/server.go

```diff
@@ -97,7 +100,7 @@ func init() {
 	viper.BindPFlag("server.address", serverCmd.Flags().Lookup("address"))
 
 	serverCmd.Flags().StringSlice("backend-mode",
-		[]string{mapper.ModeCRD, mapper.ModeConfigMap, mapper.ModeFile},
+		[]string{mapper.ModeMountedConfigMap},
```

Comment:
We should mention in the document that if you don't define the MountedConfigMap along with others in the flags it will not be part of chain and whatever customer provides will be the only mode available.

Its kind of now the default behavior and will gets overridden. 

I was thinking if we would like to have this mode always available in the server and if customer provides other chain that would come first in chain.

Reply:
I've added a call-out for this.
```
Note that when setting a single backend, the server will *only* source from
that one and ignore the others even if they exist. For example, with
`--backend-mode=CRD`, the server will *only* source from `IAMIdentityMappings`
and ignore the mounted file and EKS ConfigMap.
```

> I was thinking if we would like to have this mode always available in the server and if customer provides other chain that would come first in chain.

IMO we should see if these changes are sufficient, and if not we should add this as a final fallback.